src/styles/_base.scss

```diff
@@ -39,6 +39,7 @@
     --font-size-tiny: 0.75rem;          // 12px
     --font-size-small: 0.875rem;        // 14px
     --font-size-normal: 1.125rem;       // 18px
+	--font-size-medium-xsm: 1.125rem;   // 18px
```

Comment:
both font-size-normal and font-size-medium-xsm has same value. Use only one of them

Reply:
also spacing